app/Http/Resources/Settings/RecoveryCode/RecoveryCode.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Resources\Settings\RecoveryCode;
+
+use Illuminate\Http\Resources\Json\Resource;
+
+class RecoveryCode extends Resource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'object' => 'recoverycode',
+            'recovery' => $this->recovery,
+            'used' => (bool) $this->used,
```

Comment:
This is meant to be used in the API at some point right? Why don't you include the account information like the other resources then?

Reply:
OK the resource is not useful, I have removed it